telegram/_bot.py

```diff
@@ -241,6 +243,39 @@ def __init__(
             HTTPXRequest() if request is None else request,
         )
 
+        # this section is about raising an error when using HTTP/2 and connect to a local bot api
```

Comment:
```suggestion
        # this section is about issuing a warning when using HTTP/2 and connect to a local bot api
```

Please replace "error" with "warning" in this whole dif block :)